website/content/en/docs/migration/v0.19.0.md

```diff
@@ -0,0 +1,41 @@
+---
+title: v0.19.0
+weight: 999981000
+---
+
+## This new change will fix the ansible variables that are getting mangled
+
+The input variables with special words such as IP,HTTP and URL in the string will get converted correctly to snake_case  with all the special word(s) not being split by `_` (the special word can be plural as well).
+For example:
+
+  egressIPs                    -->  egress_ips
+
+  URLsegressIPsEgressHTTPs     -->  _urls_egress_ips_egress_https
+
+_See [#3265](https://github.com/operator-framework/operator-sdk/pull/3265) for more details._
```

Comment:
Can we improve the header and text a bit to clarify what the breaking change is, how existing users might be affected, and what affected users would need to change?

If this is not a quick fix, it can be non-blocking, but I would want to see a follow-up after the release that fixes this in both the `v0.19.x` and `master` branch.